index-el.html

```diff
@@ -645,7 +645,7 @@ <h2>Το Ανθρώπινο Κόστος</h2>
     </p>
 
     <p>
-      Αν αρκετοί άνθρωποι φορέσουν μάσκες, μπορούμε να φτάσουμε το τέλος του λόγου, όπου η πιθανότητα μόλυνσης είναι μηδενική.
+      Αν αρκετοί άνθρωποι φορέσουν μάσκες, μπορούμε να φτάσουμε το τέλος του λόφου, όπου η πιθανότητα μόλυνσης είναι μηδενική.
       Έτσι οι μάσκες μπορούν να σταματήσουν μια επιδημία.
```

Comment:
In general, I would change every 'επιδημία' with the word 'πανδημία', as this is one of the strongest arguments of people who refuse to wear them

Reply:
see comment above.